src-built-in/components/advancedAppCatalog/src/stores/storeActions.js

```diff
@@ -112,60 +111,38 @@ function _addActiveTag(tag) {
  * @param {string} tag The name of the tag
  */
 function _removeActiveTag(tag) {
-
-	let { activeTags, apps } = data;
-
-	let newActiveTags = activeTags.filter((currentTag) => {
+	data.activeTags = data.activeTags.filter((currentTag) => {
 		return currentTag !== tag;
 	});
 
-	let newApps = apps.filter((app) => {
-		for (let i = 0; i < newActiveTags.length; i++) {
-			let tag = activeTags[i].trim();
-			if (app.tags.includes(tag)) {
-				return true;
-			}
-		}
-	});
-
-	getStore().setValues([{
-		field: "activeTags",
-		value: newActiveTags
-	}, {
-		field: "filteredApps",
-		value: newApps
-	}]);
+	getStore().setValue({ field: "activeTags", value: data.activeTags	});
+	filterApps();
 }
 
 function _refreshTags() {
```

Comment:
If all this is now doing is calling `filterApps` why don't we just update all calls to this to go straight to that and delete this?

Reply:
Reasonable. I'll take it out.